proto/viam/app/data/v1/data.proto

```diff
@@ -378,3 +381,11 @@ message BoundingBoxLabelsByFilterRequest {
 message BoundingBoxLabelsByFilterResponse {
   repeated string labels = 1;
 }
+
+message GetDatabaseConnectionRequest {
+  string org_id = 1;
```

Comment:
```suggestion
  string organization_id = 1;
```
I just remembered that we recently changed all of our APIs to say "organization_id" rather than "org_id", so let's change that here.